charts/dapr/crds/components.yaml

```diff
@@ -15,56 +15,65 @@ spec:
         description: Component describes an Dapr component type.
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation
-              of an object. Servers should convert recognized schemas to the latest
-              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: The version of the Dapr (and Kubernetes if applicable) API you are calling.
```

Comment:
For Dapr this doesn't apply only to K8s and we should remove K8s specific verbiage.

Reply:
The intent would be to run any new descriptions past our docs experts as we'd want them to align with what's already in the docs (which they do not, currently, due to the difference in audience between Dapr developers and Dapr users).

Reply:
This can be referenced from here https://docs.dapr.io/reference/resource-specs/component-schema/ which is where this is defined in the docs (certainly may not be complete). Given that Dapr is platform agnostic, we stay away from the term CRD in the docs ( there are places this needs to be removed) and use the term "resource spec" or just "resource". We have to be attentive to ensuring that nothing ties Dapr to Kubernetes as the only platform.